scope/src/utils.c

```diff
@@ -36,6 +36,10 @@
 #ifdef G_OS_UNIX
 #include <fcntl.h>
 
+/* The maximum length of an expression for evaluating the value.
+   (Including the string terminator '\0') */
+#define SCOPE_MAX_EVALUATE_EXPR_LENGTH 256
```

Comment:
Is it intended that the macro definition is within the `#ifdef G_OS_UNIX` block?
This breaks the Windows builds: https://nightly.geany.org/win32/build_win32_plugins_stderr.log as it used unconditionally later on.

I assume the macro definition should be outside any OS guards?

Reply:
@eht16: yes - sorry for the hassle. Fixed in PR #885.

Reply:
@eht16: I think I have asked this before but would it be possible to add a Windows build to Travis CI?

Reply:
Thanks for #885.
Regarding Windows Travis builds: my comment https://github.com/geany/geany-plugins/pull/787#issuecomment-438838305 is still true. It is possible just needs some work and probably lots of trial-and-error.

Reply:
> If someone wants to start, I can provide the necessary GTK stack (i.e. the files we use for the nightly builds).

I added my own geany-plugins repo to Travis CI so I could start there without breaking Travis builds at geany for a probably longer time. I will create a branch for it. Yes - as offered in your previous comment - please provide the files for the nightly builds.

Reply:
Here we go:
https://download.geany.org/contrib/gtk/gtk%2B-bundle_2.24-plugins-dependencies.tar.gz
This is a tarball with GTK2 Windows binaries as well as available libraries needed by several plugins. Those libraries are currently used for the nightly builds.
As a start, I hacked a little build script: https://www.geany.org/p/ZnQOw/
This is not nice but should show how it works. I guess https://github.com/geany/geany/blob/master/scripts/cross-build-mingw.sh could be a good base to start with and just adjust it for G-P.